projects/plugins/jetpack/modules/videopress/class.jetpack-videopress.php

```diff
@@ -58,6 +58,37 @@ public function on_init() {
 
 		VideoPress_Scheduler::init();
 		VideoPress_XMLRPC::init();
+
+		if ( $this->is_videopress_enabled() ) {
+			add_action( 'admin_notices', array( $this, 'media_new_page_admin_notice' ) );
+		}
+	}
+
+	/**
+	 * The media-new.php page isn't supported for uploading to VideoPress.
+	 *
+	 * There is either a technical reason for this (bulk uploader isn't overridable),
+	 * or it is an intentional way to give site owners an option for uploading videos that bypass VideoPress.
+	 */
+	public function media_new_page_admin_notice() {
+		global $pagenow;
+
+		if ( 'media-new.php' === $pagenow ) {
+			echo '<div class="notice notice-warning is-dismissible">' .
+					'<p>' .
+					wp_kses(
```

Comment:
```suggestion
wp_kses(
						sprintf(
							/* translators: %s is the url to the Media Library */
							__( 'VideoPress uploads are not supported here. To upload to VideoPress, add your videos from the <a href="%s">Media Library</a> or the block editor using the <code>/video</code> block.', 'jetpack' ),
							esc_url( admin_url( 'upload.php' ) )
						),
						array(
							'a' => array( 'href' => array() ),
							'code' => array(),
						)
					) .
```
Adding a `code` around the `/video` suggestion looks kinda nice?

<img width="862" alt="Screen Shot 2021-10-04 at 9 36 21 AM" src="https://user-images.githubusercontent.com/789137/135880802-430226ef-3562-4233-aba0-7b35a556e868.png">

Reply:
I don't know if the shortcut for the block picker is widely used. Can we just call it the Video block?

Reply:
> Can we just call it the Video block?

Works for me!

Reply:
Just noticed this comment now... I'll revert the styling I added around `/video`.